jdaviz/configs/default/plugins/plot_options/plot_options.vue

```diff
@@ -365,40 +366,65 @@
       ></v-text-field>
     </glue-state-sync-wrapper>
 
-    <div v-if="stretch_function_sync.in_subscribed_states">
-      <v-row>
-        <v-text-field
-            ref="stretch_hist_nbins"
-            label="Number of Bins"
-            v-model.number="stretch_hist_nbins"
-            type="number"
-            hint="The amount of bins used in the histogram."
-            persistent-hint
-            :rules="[() => stretch_hist_nbins !== '' || 'This field is required',
-                     () => stretch_hist_nbins > 0 || 'Number of Bins must be greater than zero']"
-        ></v-text-field>
-      </v-row>
-      <v-row>
-        <!-- z-index to ensure on top of the jupyter widget with negative margin-top -->
-        <v-switch
-          v-model="stretch_hist_zoom_limits"
-          class="hide-input"
-          label="Limit histogram to current zoom limits"
-          style="z-index: 1"
-        ></v-switch>
-      </v-row>
+    <div v-if="stretch_function_sync.in_subscribed_states && !multiselect">
+      <jupyter-widget :widget="stretch_histogram_widget"/>
       <v-row>
-        <v-switch
-          v-model="stretch_curve_visible"
-          class="hide-input"
-          label="Show stretch function curve"
-          style="z-index: 1"
-        ></v-switch>
-        <!-- NOTE: height defined here should match that in the custom CSS rules
-             below for the bqplot class -->
+        <v-expansion-panels accordion>
+          <v-expansion-panel>
+            <v-expansion-panel-header v-slot="{ open }">
+              <span style="padding: 6px">More Stretch Options</span>
+            </v-expansion-panel-header>
+            <v-expansion-panel-content>

```

Comment:
```suggestion
            <v-expansion-panel-content style="padding: 10px;">

```

Reply:
you caught me!... we probably should do this across the various other instances as well.  Let me see if we can set a general css rule here, or if not, open a ticket to do that.

Reply:
See https://github.com/spacetelescope/jdaviz/pull/2540